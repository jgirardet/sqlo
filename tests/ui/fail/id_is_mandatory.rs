#[derive(sqlo::Sqlo)]
struct Bla {
    un: String,
}

fn main() {}
