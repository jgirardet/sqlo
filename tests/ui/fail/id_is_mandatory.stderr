error: Sqlo should have one field with attirbute `primary_key` or an `id` field!
 --> tests/ui/fail/id_is_mandatory.rs:2:8
  |
2 | struct Bla {
  |        ^^^
